rasa_core/evaluate.py

```diff
@@ -60,7 +60,11 @@ def create_argument_parser():
             type=str,
             default="failed_stories.txt",
             help="output path for the failed stories")
-
+    parser.add_argument(
```

Comment:
this has already been added in #966 as `--fail_on_prediction_errors `

Reply:
👌